config/kraft/controller.properties

```diff
@@ -26,13 +26,13 @@ process.roles=controller
 # The node id associated with this instance's roles
 node.id=1
 
-# The connect string for the controller quorum
```

Comment:
We still support static quorum, and will for a while. I would suggest commenting the line with `controller.quorum.voters` out, and adding a comment saying that you should comment out either `controller.quorum.voters` or `controller.quorum.bootstrap.servers`, depending on whether you want to use a KIP-853 dynamic quorum or not.

(By default, the uncommented one can be `controller.quorum.bootstrap.servers`, so that more people try the new thing.)

Reply:
I guess that depends on the intent for this configuration file. My interpretation is this is use to quickly create a simple cluster for quickstart or other similar cases. If that is the use cases, the user doesn't care if they are using kraft.version 0 or kraft.version 1. What do you think?

Reply:
For now, let's document it. We can change it later if needed.